src/content/docs/extensions/json.mdx

```diff
@@ -548,34 +548,44 @@ Output:
 ```
 
 ```cypher
-RETURN to_json({duck : 42});
+RETURN to_json({kuzu : 42});
 ```
 Output:
 ```json
-{"duck":42}
+{"kuzu":42}
 ```
 
 ```cypher
-RETURN to_json(map(['duck'],[42]));
+RETURN to_json(map(['kuzu'],[42]));
 ```
 Output:
 ```json
-{"duck":42}
+{"kuzu":42}
 ```
 
 ### `json_merge_patch`
 
 Signature: `JSON, JSON -> JSON`
 
+`json_merge_patch(json, json)`
+
 Merges two `JSON` documents. Applies [RFC 7386](https://datatracker.ietf.org/doc/html/rfc7386)
 
+```cypher
+RETURN json_merge_patch('{"kuzu": 114}', '{"goose": 514}');
+```
+Output:
+```json
+{"kuzu":114,"goose":514} 
+```
+
 ### `json_extract`
 
 Signatures: `JSON, STRING -> JSON`, `JSON, INTEGER -> JSON`
 
 `json_extract(json, path)`
 
-Extracts `JSON` from `json` at the given `path`.  `STRING` `path` is delimited by `'/'`, while `INTEGER` path is only used to index `JSON arrays`. If `path` is a `LIST`, the result will be a `LIST` of `JSON`.
+Extracts `JSON` from `json` at the given `path`.  `STRING` `path` is delimited by `'/'`, while integer path is only used to index JSON arrays. If `path` is a `LIST`, the result will be a `LIST` of JSON.
```

Comment:
@acquamarin this part isn't clear - what does "integer path" even mean? Why would the user provide a path which is an integer?

Reply:
the integer path means the index, let me try rephrase that

Reply:
re paraphrase this a little